### ISSUE TITLE
Some Xsl tests don't work on non-coreclr platforms

### ISSUE THREAD

**Issue by krwq** [dotnet/corefx#10305]
Projects affected:
- XsltCompiler
- System.Xml.Xsl.XslCompiledTransformApi.Tests
- System.Xml.Xsl.XslTransformApi.Tests (partially)
- XsltScenarios.Tests (partially)
- check all xml projects + Serialization.Xml + Serialization.Json test projects

Two issues:
- System.Runtime.Loader should not be used by project (it's making the project need to use netstandard1.5 instead of 1.3)
- few of the libraries won't work on UAP: System.Diagnostics.Process

Basically: fix project.json to do what other test projects do and fix the compile/restore errors.

**Comment by karelz** [comment]
@krwq why is it in netstandard2.0 if it does not affect CoreCLR?

**Comment by krwq** [comment]
I believe this bug is not relevant anymore
